service/src/main/java/com/commercetools/util/HttpRequestUtil.java

```diff
@@ -39,6 +40,8 @@
  * <li>retries on connections exceptions up to 3 times, if request has not been sent yet
  * (see {@link DefaultHttpRequestRetryHandler#isRequestSentRetryEnabled()}
  * and {@link #HTTP_REQUEST_RETRY_ON_SOCKET_TIMEOUT})</li>
+ * <li>connections pool is 200 connections, up to 20 per route (see {@link #CONNECTION_MAX_TOTAL}
```

Comment:
Maybe makes sense to specify the reason for these numbers? 200 and 20?

Reply:
I'd also like to know, but i have not found any explicit recommendations for the pool sizes. This value is "inherited" from Unirest, but in Unirest it is also not explained why 200/20 is a good size.
We will be using this numbers as long as we don't face problems with out of memory or out of connections.

Reply:
ok, could you please add this there :)? At least if problems where faced, a future dev would see that there where no specific reason for them in the first place.

Reply:
5e06833e